tests/test_handler.py

```diff
@@ -55,7 +55,7 @@ def test_stop_test_db_nothing_to_stop(self, mock_boto):
         assert result['message'] == 'Stopped the test db: False'
 
     @mock.patch.dict('src.utils.os.environ', mock_env_vars)
-    @mock.patch('src.utils.boto3.client', autospec=True)
+    @mock.patch('src.utils.boto3', autospec=True)
```

Comment:
I'm not confident mocking boto3 here is the best way to go here. It may be more useful to mock the function calls made by the handlers instead.